projects/sandbox/pyproject.toml

```diff
@@ -79,7 +79,7 @@ reference_frequency = "${base.reference_frequency}"
 datadir = "${base.datadir}"
 logdir = "${base.logdir}"
 n_samples = 10000
```

Comment:
Could you swap this to `num_signals`? I changed the variable name in the project but forgot to update it here.